lib/pure/random.nim

```diff
@@ -656,4 +656,15 @@ when not defined(nimscript) and not defined(standalone):
       let now = times.getTime()
       randomize(convert(Seconds, Nanoseconds, now.toUnix) + now.nanosecond)
 
+proc sampleBuffer*[T](r: var Rand, buffer: var openArray[T]; alphabet: set[T]) {.inline, since: (1, 5).} =
+  ## Populates `buffer` with random samples from `alphabet`.
+  ## Can be used to generate random ASCII URL-Safe strings of specified length and chars.
```

Comment:
this line now feels a bit out of place unless we can add an example showing how to do that, eg by exporting `cb64safeVM` in base64; can be done in future work though

Reply:
`cb64safeVM` is a bad name for an exported symbol though, and in any case this would require a doc comment, so I recommend deferring the exporting of `cb64safeVM` to a PR that can come after this one?

Reply:
@timotheecour Check the latest change, I think is the proper way.

Reply:
excellent